Content.Client/GameObjects/Components/Items/ClientHandsComponent.cs

```diff
@@ -6,6 +6,7 @@
 using Content.Client.Interfaces.GameObjects;
 using Content.Client.UserInterface;
 using Content.Shared.GameObjects;
+using Mono.Cecil;
```

Comment:
Is this not bringing in a Mono-only dependency?

Reply:
AFAIK, it's not only mono-only. But I don't know how the hell that one got in this file. Removed now though.